packages/design-system-react-native/src/primitives/Blockies/Blockies.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from '@storybook/react-native';
+import { View } from 'react-native';
+
+import Blockies from './Blockies';
+import type { BlockiesProps } from './Blockies.types';
+
+const meta: Meta<BlockiesProps> = {
+  title: 'Primitives/Blockies',
+  component: Blockies,
+  argTypes: {
+    size: {
+      control: 'number',
+    },
```

Comment:
non-blocking: do we need an address control here?

Reply:
I dont think we do but don't have a strong preference. lmk if you want me to add it